cmd/template/dbdriver/files/service/redis.tmpl

```diff
@@ -2,11 +2,12 @@ package database
 
 import (
 	"context"
-	"crypto/tls"
```

Comment:
Why did you remove the 'crypto/tls' package? You should also comment it out because we are leaving the TLS config in the code